app/src/main/java/com/example/imageEditor/ui/home/adapter/CollectionAdapter.kt

```diff
@@ -0,0 +1,114 @@
+package com.example.imageEditor.ui.home.adapter
+
+import android.util.SparseIntArray
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.core.text.HtmlCompat
+import androidx.core.util.containsKey
+import androidx.core.util.set
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.PagerSnapHelper
+import androidx.recyclerview.widget.RecyclerView
+import com.example.imageEditor.R
+import com.example.imageEditor.databinding.ItemCollectionBinding
+import com.example.imageEditor.model.CollectionModel
+import com.example.imageEditor.utils.displayImage
+
+class CollectionAdapter : ListAdapter<CollectionModel, CollectionAdapter.ViewHolder>(
+    object : DiffUtil.ItemCallback<CollectionModel>() {
+        override fun areItemsTheSame(
+            oldItem: CollectionModel,
+            newItem: CollectionModel,
+        ): Boolean {
+            return oldItem.id == newItem.id
+        }
+
+        override fun areContentsTheSame(
+            oldItem: CollectionModel,
+            newItem: CollectionModel,
+        ): Boolean {
+            return oldItem.id == newItem.id
+        }
+    },
+) {
+    private val childRecyclerViewState = SparseIntArray()
+
+    class ViewHolder(private val binding: ItemCollectionBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun bindView(
+            item: CollectionModel,
+            childRecyclerViewState: SparseIntArray,
+            index: Int,
+        ) {
+            binding.imgUser.displayImage(item.user.profileImage.small)
+            binding.tvUserName.text = item.user.username
+            binding.tvLocation.text = item.user.location
+
+            val adapter = ImageAdapter(binding.root.context, item.previewPhotos)
+            binding.recycleViewImg.adapter = adapter
+
+            binding.tvLikes.text =
+                binding.root.context.getString(
+                    R.string.liked_by_others,
+                    item.coverPhoto.likes.toString(),
+                )
+
+            item.description?.let {
+                val descriptionHtml = "<b>${item.user.username}</b> ${item.description}"
+                binding.tvDescription.text =
+                    HtmlCompat.fromHtml(descriptionHtml, HtmlCompat.FROM_HTML_MODE_LEGACY)
+            }
```

Comment:
```suggestion
            item.description?.let {
                val descriptionHtml = "<b>${item.user.username}</b> ${item.description}"
                binding.tvDescription.text =
                    HtmlCompat.fromHtml(descriptionHtml, HtmlCompat.FROM_HTML_MODE_LEGACY)
            }
```
Khai báo thành 1 hàm ``getDescriptionHTML()`` bên trong class ``CollectionModel`` rồi sử dụng